skimage/morphology/convex_hull.py

```diff
@@ -23,6 +27,14 @@ def convex_hull_image(image):
     ----------
     image : (M, N) array
```

Comment:
The shape pattern should be updated (not (M, N) anymore)

Reply:
Remember this comment.